tests/features/asset_distribution/add_solution.feature

```diff
@@ -0,0 +1,53 @@
+@api
+Feature: "Add asset distribution" visibility options.
+  In order to manage distributions
+  As a moderator
+  I need to be able to add "Asset distribution" rdf entities through UI.
+
+  Scenario: "Add distribution" button should only be shown to moderators.
+    Given the following collection:
+      | name | Asset Distribution Test                  |
+      | uri  | https://a.distribution/solution/add/test |
+      | logo | logo.png                                 |
+    And the following solution:
+      | name              | Asset random name                        |
+      | uri               | http://joinup.eu/ad/solution/api/bar     |
+      | description       | Some reusable random description         |
+      | elibrary creation | 1                                        |
+      | groups audience   | https://a.distribution/solution/add/test |
+
+    When I am logged in as a "moderator"
+    And I go to the homepage of the "Asset random name" solution
```

Comment:
I just tried it manually and it is impossible to reach the solution starting from the homepage.

1. There is no link to the collection overview on the homepage. This is fixed in ISAICP-2457 which has been accepted in develop recently.
2. After I merged in the latest develop and saw the link to the collection overview. However when I clicked it I got an error message: "Notice: Undefined index: server in Drupal\search_api_db\Plugin\search_api\backend\Database->getIndexDbInfo() (line 2460 of modules/contrib/search_api/search_api_db/src/Plugin/search_api/backend/Database.php)." I will try again after a full rebuild and reinstall.
2. The list of collections is empty. This will be fixed in ISAICP-2458.
3. When I finally make it to the collection page by typing in the URL manually, there are no links to the available solutions. There is even no evidence at all that this collection has solutions associated with it.
4. When I go to the solution page by typing in the URL manually, I don't see any evidence that there are distributions associated with the solution.

So in short, there are 4 critical issues blocking